src/main/java/com/eternalcode/discordapp/Embeds.java

```diff
@@ -0,0 +1,26 @@
+package com.eternalcode.discordapp;
+
+import net.dv8tion.jda.api.EmbedBuilder;
+
+import java.awt.*;
+import java.time.Instant;
+
+public class Embeds {
```

Comment:
No to ładnie nie wygląda, ale jak już jest public to daj final te fieldy w klasie.

Reply:
A jak to inaczej zrobić? Chce żeby ustawiało wszędzie defaultowy footer itp, póki go nie zmodyfikuje

Reply:
Ten footer się gdzieś zapisuje czy to tylko do wyłączenia jest?

Reply:
???

Reply:
Po prostu wykorzystuje to wtedy tak:

```java
            MessageEmbed successSend = new Embeds().success
                    .setDescription("Embed sent successfully")
                    .build();
```

Póki nie wywołam MessageEmbed#setFooter, to będzie footer z tej klasy

Reply:
No czy to do konfiguracji leci?

Reply:
nie